src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Network Analysis/FindServiceArea/FindServiceArea.xaml.cs

```diff
@@ -0,0 +1,239 @@
+            ServiceAreaTask serviceAreaTask = await ServiceAreaTask.CreateAsync(_serviceAreaUri);
+
+            // An object that defines parameters for solving a service area task.
+            ServiceAreaParameters serviceAreaParameters;
+
+            // Store the default parameters for the service area in an object.
+            serviceAreaParameters = await serviceAreaTask.CreateDefaultParametersAsync();
+
+            // Add impedance cutoffs for facilities (drive time minutes).
+            serviceAreaParameters.DefaultImpedanceCutoffs.Add(2.0);
+            serviceAreaParameters.DefaultImpedanceCutoffs.Add(5.0);
+
+            // Set the level of detail for the polygons.
+            serviceAreaParameters.PolygonDetail = ServiceAreaPolygonDetail.High;
+
+            // Get a list of the barriers from the graphics overlay.
+            List<PolylineBarrier> polylineBarriers = (from g in allGraphics
+                                                      where (string)g.Attributes["Type"] == "Barrier"
+                                                      select new PolylineBarrier((Polyline)g.Geometry)).ToList();
+
+            // Add the barriers to the service area parameters.
+            serviceAreaParameters.SetPolylineBarriers(polylineBarriers);
+
+            // Update the parameters to include all of the placed facilities.
+            serviceAreaParameters.SetFacilities(serviceAreaFacilities);
+
+            // Clear existing graphics for service areas.
+            foreach (Graphic g in allGraphics.ToList())
+            {
+                // Check if the graphic g is a service area.
+                if ((string)g.Attributes["Type"] == "ServiceArea")
+                {
+                    allGraphics.Remove(g);
+                }
+            }
+
+            try
+            {
+                // Solve for the service area of the facilities.
+                ServiceAreaResult result = await serviceAreaTask.SolveServiceAreaAsync(serviceAreaParameters);
+
+                // Loop over each facility.
+                for (int i = 0; i < serviceAreaFacilities.Count; i++)
+                {
+                    // Create list of polygons from a service facility.
+                    List<ServiceAreaPolygon> polygons = result.GetResultPolygons(i).ToList();
+
+                    // Symbol for the outline of the service areas.
+                    SimpleLineSymbol serviceOutline = new SimpleLineSymbol(SimpleLineSymbolStyle.Solid, System.Drawing.Color.Black, 3.0f);
+
+                    // Create a list of fill symbols for the polygons.
+                    List<SimpleFillSymbol> fillSymbols = new List<SimpleFillSymbol>();
+                    fillSymbols.Add(new SimpleFillSymbol(SimpleFillSymbolStyle.Solid, System.Drawing.Color.FromArgb(70, 255, 0, 0), serviceOutline));
+                    fillSymbols.Add(new SimpleFillSymbol(SimpleFillSymbolStyle.Solid, System.Drawing.Color.FromArgb(70, 255, 165, 0), serviceOutline));
```

Comment:
I think the result polygons look much better with a thinner/lighter outline (darkgray and/or 1.0 width?).